checker/bin-devel/test-cf-inference.sh

```diff
@@ -26,4 +26,5 @@ BRANCH=`/tmp/plume-scripts/git-find-branch ${REPO} ${CI_BRANCH}`
 
 export AFU=`readlink -f ${AFU:-../annotation-tools/annotation-file-utilities}`
 export PATH=$AFU/scripts:$PATH
+(cd ../checker-framework-inference && ./.travis-build-without-test.sh)
```

Comment:
Instead of these two lines, why not just execute `.travis-build.sh` here?

Reply:
Done. Thanks!